packages/redux/src/returns/reducer.js

```diff
@@ -56,6 +59,8 @@ const id = (state = INITIAL_STATE.id, action = {}) => {
       return action.payload.result;
     case actionTypes.RESET_RETURN:
       return INITIAL_STATE.id;
+    case LOGOUT_SUCCESS:
+      return INITIAL_STATE.id;
```

Comment:
this could be added to the previous case instead of adding a new one with the same info, i.e:

```
case actionTypes.RESET_RETURN
case LOGOUT_SUCCESS:
   INITIAL_STATE.id;
```

do you agree?

Reply:
Make sense, I´ll change it